### PR TITLE
Harvest

### PR DESCRIPTION
**Pull request by Jonathan-dehaay**
Harvest view.
Create loads, with quantity, nature and storages.
Solved some issues.

### CODE REVIEW

File: Controllers/AddHarvest.swift [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+          storagesNames.append(storage.name!)
+        }
+      }
+    } catch let error as NSError {
+      print("Could not fetch. \(error), \(error.userInfo)")
+    }
+    return storagesNames
+  }
+
+  func createSampleStorage() {
```
[the remark this answers is not in the document]
Comment (Jonathan-dehaay) [reply Controllers/AddHarvest.swift]: No problem, when storage creation will be implemented, it'll be removed.

File: Controllers/AddIrrigation.swift [hunk 2]
```diff
@@ -34,20 +34,39 @@ extension AddInterventionViewController: UITextFieldDelegate, CustomPickerViewPr
   }
 
   // MARK: - Picker view
-
+
```
Comment (TontonBuck) [inline Controllers/AddIrrigation.swift]: Bad indentation

File: Controllers/AddWeatherToIntervention.swift [hunk 3]
```diff
@@ -13,6 +13,7 @@ extension AddInterventionViewController {
 
   // MARK: - Initialization
 
+
```
Comment (TontonBuck) [inline Controllers/AddWeatherToIntervention.swift]: Trailing line

File: Resources/Translations/Base.lproj/Localizable.strings [hunk 4]
```diff
@@ -6,6 +6,7 @@
   Copyright © 2018 Ekylibre. All rights reserved.
 */
 
+
```
Comment (TontonBuck) [inline Resources/Translations/Base.lproj/Localizable.strings]: Trailing line

File: Resources/Translations/Base.lproj/Localizable.strings [hunk 5]
```diff
@@ -413,6 +417,8 @@
 "last_name" = "Nom de famille";
 "equipment_name_not_available" = "Ce nom d'équipement est déjà utilisé. Veuillez en choisir un autre";
 "equipment_name_is_empty" = "Le nom de l'équipement ne peut être vide";
+"person_name_not_available" = "Ce nom est déjà utilisé. Veuillez en choisir un autre";
```
Comment (TontonBuck) [inline Resources/Translations/Base.lproj/Localizable.strings]: Added different translation in Localization branch

File: Resources/Translations/Base.lproj/Localizable.strings [hunk 6]
```diff
@@ -413,6 +417,8 @@
...
+"person_name_is_empty" = "Le prénom de la personne ne peut être vide";
```
Comment (TontonBuck) [inline Resources/Translations/Base.lproj/Localizable.strings]: Added different translation in Localization branch

File: Views/CustomPickerView.swift [hunk 7]
```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+
```
Comment (TontonBuck) [inline Views/CustomPickerView.swift]: Trailing line

File: Views/Storyboards/Base.lproj/Cartfile [hunk 8]
```diff
@@ -0,0 +1,2 @@
+github "p2/OAuth2" >= 4.0
```
Comment (TontonBuck) [inline Views/Storyboards/Base.lproj/Cartfile]: Duplicated file

File: Views/Storyboards/fr.lproj/Main.strings [hunk 9]
```diff
@@ -86,7 +86,7 @@
 /* Class = "UILabel"; text = "km/h"; ObjectID = "W1a-qv-ooU"; */
 "W1a-qv-ooU.text" = "km/h";
 
-/* Class = "UIButton"; normalTitle = "REGISTER AN INTERVENTION"; ObjectID = "XRU-Wr-aoY"; */
+/* Class = "UIButton"; normalTitle = "ALREADY DONE"; ObjectID = "XRU-Wr-aoY"; */
```
Comment (TontonBuck) [inline Views/Storyboards/fr.lproj/Main.strings]: Should be "REGISTER AN INTERVENTION"

**Pull request merged**